protocols/PipelineFinished.sh

```diff
@@ -0,0 +1,21 @@
+#MOLGENIS walltime=05:59:00 mem=10gb ppn=6
+
+#string intermediateDir
+#string Project
+#string logsDir
+#string runID
+
+set -e
+set -u
+
+# Touch log file for NGS_Automated for starting copying rawdata to PRM
+
+if [ -f "${logsDir}/${Project}/${Project}.${runID}.AGCT.started" ]
```

Comment:
deze hele file is bijna overbodig, started naar finished hoeft niet echt.
Het enige wat we nodig zijn is een .finished file

Kun je de hele if else gewoon weghalen en dan dit er voor in de plaats zetten:
touch "${logsDir}/${Project}/${runID}.AGCT.finished"

Die naam van de file strookt dan weer meer